x/multi-staking/keeper/invartiants_test.go

```diff
@@ -33,9 +37,9 @@ func (suite *KeeperTestSuite) TestModuleAccountInvariants() {
 			name: "Success Edit Validator",
 			malleate: func() {
 				suite.ctx = suite.ctx.WithBlockHeader(tmproto.Header{Time: time.Now()})
-				newRate := sdk.MustNewDecFromStr("0.03")
-				newMinSelfDelegation := sdk.NewInt(300)
-				editMsg := stakingtypes.NewMsgEditValidator(valAddr, stakingtypes.Description{
+				newRate := math.LegacyMustNewDecFromStr("0.03")
+				newMinSelfDelegation := math.NewInt(300)
+				editMsg := stakingtypes.NewMsgEditValidator(valAddr.String(), stakingtypes.Description{
```

Comment:
`NewMsgCreateValidator`, `NewValidator`, `NewMsgCancelUnbondingDelegation`, `NewMsgUndelegate`, `NewMsgBeginRedelegate`, `NewMsgDelegate` and `NewMsgEditValidator`  takes a string instead of `sdk.ValAddress` or `sdk.